### PR TITLE
Add videojs-annotation-comments plugin and put it behind a feature flag

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
https://trello.com/c/YlaO3AvR/25-annotation-research

#### What's this PR do?
Adds the videojs-annotation-comments plugin and put it behind a feature flag so that we can try it out

#### How should this be manually tested?
Nothing should break by default. Set `FEATURE_VIDEOJS_ANNOTATIONS=True` and you should see a simple annotations UI when you look at videos using videojs. You can make changes but they will not be persisted anywhere and will go away when the browser refreshes.

### CODE REVIEW

File: ui/views.py [hunk 1]
```diff
@@ -70,8 +71,10 @@ def default_js_settings(request):
         ),
         "support_email_address": settings.EMAIL_SUPPORT,
         "ga_dimension_camera": settings.GA_DIMENSION_CAMERA,
+        "video_annotations": video_annotations_feature_flag,
```
Comment (gsidebo) [inline ui/views.py]: Do we need this redundancy? Can we just check for `FEATURES["VIDEOJS_ANNOTATIONS"]` in the template?

File: ui/templates/base.html [hunk 2]
```diff
@@ -11,7 +11,10 @@
     {% load raven %}
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
-    <script type="text/javascript">
+      {% if video_annotations %}
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```
Comment (gsidebo) [inline ui/templates/base.html]: I'm guessing this means that this `videojs-annotation-comments` plugin requires jQuery? Could you not add this dependency as a `require` call in `static/js/lib/video.js`?

**Comment by noisecapella** [comment]
Comments should be addressed. The template variable was actually set as a JS variable so the jquery link was never inserted, which is probably why it didn't work for you. Can you try it again?

File: ui/templates/base.html [hunk 3]
```diff
@@ -11,7 +11,7 @@
     {% load raven %}
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
-    <script type="text/javascript">
+      <script type="text/javascript">
```
Comment (gsidebo) [inline ui/templates/base.html]: Seems like an unintentional change

**Pull request merged**